plugins/jira/skills/status-analysis/scripts/gather_status_data.py

```
@@ -1216,6 +1222,7 @@ def _build_manifest(
 
             # Build issue data
             assignee = fields.get("assignee") or {}
+            labels = [l for l in fields.get("labels", []) if isinstance(l, str)]
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)